src/ExtensionGallery/Startup.cs

```diff
@@ -1,109 +1,63 @@
 ﻿using System;
-using Microsoft.AspNet.Http;
 using System.Linq;
-using Microsoft.AspNet.Builder;
-using Microsoft.AspNet.Diagnostics;
-using Microsoft.AspNet.Hosting;
-using Microsoft.AspNet.Routing;
-using Microsoft.AspNet.StaticFiles;
-using Microsoft.Framework.Configuration;
-using Microsoft.Framework.DependencyInjection;
-using Microsoft.Framework.Logging;
-using Microsoft.Framework.Runtime;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.AspNetCore.StaticFiles;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.PlatformAbstractions;
+using Newtonsoft.Json.Serialization;
 
 namespace ExtensionGallery
 {
 	public class Startup
 	{
-		public Startup(IHostingEnvironment env, IApplicationEnvironment appEnv)
+		public Startup(IHostingEnvironment env)
 		{
             // Setup configuration sources.
-            Configuration = new ConfigurationBuilder(appEnv.ApplicationBasePath)
-				.AddJsonFile("config.json")
```

Comment:
It looked like the config wasn't being used anyway, so I just used the default config setup that the tooling generates.